jaseci_ai_kit/jaseci_ai_kit/modules/whisper/requirements.txt

```diff
@@ -0,0 +1,3 @@
+transformers>=4.25.1
+librosa>=0.9.2
+soundfile>=0.11.0
```

Comment:
I don't see soundfile in the setup.py. Do we need to add it?

Reply:
SOundfile get installed with librosa